main.cpp

```diff
@@ -10,158 +9,266 @@ video: Chapter 2 - Part 3
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+// 2)
+void variableDeclarations() {
+  // example:
+  int number =
+      2; // declaration of a variable named "number", that uses the primitive
+         // type 'int', and the variable's initial value is '2'
+
+    // Integer type
+    int myInt = 42;
+    int anotherInt = -10;
+    int thirdInt = 0;
+
+    // Float type
+    float myFloat = 3.14f;
+    float anotherFloat = 2.718f;
+    float thirdFloat = 1.0f;
+
+    // Double type
+    double myDouble = 2.71828;
+    double anotherDouble = 3.14159265359;
+    double thirdDouble = 1.61803398875;
+    
+    // Boolean type
+    bool isTrue = true;
+    bool isFalse = false;
+    bool isFlag = true;
+
+
+  ignoreUnused(number); // passing each variable declared to the ignoreUnused() function
+
+    ignoreUnused(myInt);
+    ignoreUnused(anotherInt);
+    ignoreUnused(thirdInt);
+    ignoreUnused(myFloat);
+    ignoreUnused(anotherFloat);
+    ignoreUnused(thirdFloat);
+    ignoreUnused(myDouble);
+    ignoreUnused(anotherDouble);
+    ignoreUnused(thirdDouble);
+    ignoreUnused(isTrue);
+    ignoreUnused(isFalse);
+    ignoreUnused(isFlag);
 }
 
 /*
  10 functions
  example:
  note: this example shows the result after completing steps 3-8
  */
-bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
-    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
+bool rentACar(
+    int rentalDuration,
+    int carType = 0) // function declaration with random number of arguments,
+                     // arbitrary number of arguments have default value
+
+{
+  ignoreUnused(rentalDuration, carType); // passing each function parameter to
+                                         // the ignoreUnused() function
+  return {}; // if your function returns something other than void, add 'return
+             // {};' at the end of it.
+}
+
+// 1) Function to calculate the area of a rectangle 
+double calculateRectangleArea(double length, double width) 
+{ 
+    ignoreUnused(length, width); 
+    return length * width; 
+}  
+
+// 2) Function to calculate the volume of a box 
+double calculateBoxVolume(double length, double width, double height) 
+{ 
+    ignoreUnused(length, width, height); 
+    return length * width * height; 
+} 
+
+// 3) Function to convert Fahrenheit to Celsius 
+double fahrenheitToCelsius(double fahrenheit) 
+{ 
+    ignoreUnused(fahrenheit); 
+    return (fahrenheit - 32.0) * 5.0 / 9.0; 
+} 
+
+// 4) Function to convert Celsius to Fahrenheit 
+double celsiusToFahrenheit(double celsius) 
+{ 
```

Comment:
These function names do not have verbs in them, and thus do not describe actions
- [ ] revise